roles/calico/defaults/main.yaml

```diff
@@ -7,6 +7,7 @@ calico_node_image: "quay.io/calico/node:v3.5.0"
 calico_cni_image: "quay.io/calico/cni:v3.5.0"
 calicoctl_image: "quay.io/calico/ctl:v3.5.0"
 calico_upgrade_image: "quay.io/calico/upgrade:v1.0.5"
+calico_ip_autodetection_method: "first-found"
```

Comment:
Not sure if this is a default method Calico uses, @mgleung please confirm this won't change default behaviour

Reply:
From the documention, [first-found](https://docs.projectcalico.org/v3.6/reference/node/configuration#first-found): 

> This is the default detection method. However, since this method only makes a very simplified guess, it is recommended to either configure the node with a specific IP address, or to use one of the other detection methods.

Reply:
Right, I'm just concerned that setting IP_AUTODETECTION_METHOD changes the way calico detects

Reply:
Thanks for checking guys. Setting `IP_AUTODETECTION_METHOD` to the default of `first-found` won't be treated any differently than not setting the value at all so this should be fine.